pages/index.tsx

```diff
@@ -41,7 +74,12 @@ export default function Home() {
       .catch((err) => console.log("err get", err));
   }
 
-  useEffect(initialize)
+  useEffect(initialize, [])
+  useEffect(() => {
+    localStorage.getItem("onNetwork") && setOnNetwork(localStorage.getItem("onNetwork") || 0)
```

Comment:
... don't use inline ifs. **don't make me make a lint rule for it**

switch this to something that makes actual logical sense:

```ts
setOnNetwork(localStorage.getItem("onNetwork") || 0);
setBeproStaked(localStorage.getItem('oracles') || 0);
setTotalAmount(localStorage.getItem("total") || 0);
```